Task/python/Taskfile.yaml

```
@@ -0,0 +1,53 @@
+# https://taskfile.dev
+
+version: '3'
+
+vars:
+  IMAGE_NAME: null
+  VERSION: null
+  PYTHON_VERSION: null
+
+tasks:
+  lint:
+    preconditions:
+      - sh: test `git ls-files -z -o --exclude-standard | wc -w` = "0"
+        msg: "There are untracked files, please commit before linting."
+    cmds:
+      - docker run --rm -v ${PWD}:/goat -e RUN_LOCAL=true -e EXCLUDE="/goat" seiso/goat:latest
+
+  test:
+    preconditions:
+      - sh: test `git symbolic-ref -q HEAD`
+        msg: "In detached HEAD state."
+    cmds:
+      - pipenv run pytest tests 2>&1 >/dev/null
+
+  build:
+    cmds:
+      - docker build -t {{.IMAGE_NAME}}:latest -t {{.IMAGE_NAME}}:{{.VERSION}} .
```

If we're going to be able to use `docker` again, we should enable buildkit.  It's often on by default, but some of those details are out of our control, so we can do something like:
```suggestion
      - DOCKER_BUILDKIT=1 docker build -t {{.IMAGE_NAME}}:latest -t {{.IMAGE_NAME}}:{{.VERSION}} .
```
Or something like:
```suggestion
      - docker buildx build -t {{.IMAGE_NAME}}:latest -t {{.IMAGE_NAME}}:{{.VERSION}} .
```

We should weigh the pros and cons of each and pick the best for our situation.

Also, this needs to identify if this is a release or not, and set a variable accordingly for the buildargs.  Right now, VERSION and COMMIT_HASH must be passed in as buildargs with the right information.  Details [here](https://github.com/SeisoLLC/cookiecutter-python/blob/085215423d3c66a42408cc7d2a0b0eb18b62c0f6/%7B%7Bcookiecutter.project_name%7Creplace(%22%20%22%2C%20%22%22)%7D%7D/tasks.py#L114).